core/src/main/java/com/segment/analytics/kotlin/core/Storage.kt

```diff
@@ -28,6 +29,8 @@ interface Storage {
          * is not present in payloads themselves, but is added later, such as `sentAt`, `integrations` and other json tokens.
          */
         const val MAX_BATCH_SIZE = 475000 // 475KB.
+
+        const val MAX_FILE_SIZE = 475_000
```

Comment:
Do we think these will need to be different?  Maybe good just for variable name clarity.

Reply:
no. it's not gonna be different. it's something set on backend.